Readme.adoc

```diff
@@ -73,6 +73,9 @@ testing {
 The added companion task has the pattern of `<suiteName>GenerateCucumberSuiteCompanion`, so for a suite named `functionalTest` it would be `functionalTestGenerateCucumberSuiteCompanion`.
 However, there's usually no need to call it manually since the `compile<suiteName>Java` task depends on it.
 
+To add `failIfNoTests = false` to the generated `@Suite` annotation in companions, just add second parameter `true`
+to the invocation.
```

Comment:
I'm a bit afraid that this approach to configure the companion task does not scale very well in case we'll add further parameters. I think a more idiomatic approach would be, to pass in an optional action that allows to configure some aspects.

Like:
```
testing {
    suites {
        functionalTest {
            generateCucumberSuiteCompanion(project) {
                allowEmptySuites.set(true)
            }
        }
    }
}
```

Reply:
I agree with @skuzzle that we should be able to configure it on a per-task level. This especially makes sense if the targeted use-case is about retries. Those can also be configured only on a per-task basis. Otherwise, it looks good from my side, too. Great job!

Reply:
Thanks for the hint with the gradle's Action - I was thinking about something like `Options`, but now Kotlin functions and Groovy closures are being resolved to the `Action` and applied to the generator task.
Please have another look.